src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -65,6 +68,10 @@ public class Drivetrain extends SubsystemBase {
     // Angle to align to, null for directly toward speaker
     private Double alignAngle = null;
 
+    // Latency calculation
+    private int maxSecondsStored = 1; //max number of past frames to store
```

Comment:
is this 1 second or number of past frames to store? the variable name and comment are confusing